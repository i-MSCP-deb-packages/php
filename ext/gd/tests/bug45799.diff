001+ Parse error: imagepng(): gd-png error: no colors in palette
001- Warning: imagepng(): gd-png error: no colors in palette in %s on line %d
002+  in /home/ondrej/Projects/pkg-php/php-src/ext/gd/tests/bug45799.php on line 3