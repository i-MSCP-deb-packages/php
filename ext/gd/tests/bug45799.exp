Warning: imagepng(): gd-png error: no colors in palette in %s on line %d