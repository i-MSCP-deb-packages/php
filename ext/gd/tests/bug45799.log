
---- EXPECTED OUTPUT
Warning: imagepng(): gd-png error: no colors in palette in %s on line %d
---- ACTUAL OUTPUT
Parse error: imagepng(): gd-png error: no colors in palette
 in /home/ondrej/Projects/pkg-php/php-src/ext/gd/tests/bug45799.php on line 3
---- FAILED
