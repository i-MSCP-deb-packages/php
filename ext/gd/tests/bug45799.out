Parse error: imagepng(): gd-png error: no colors in palette
 in /home/ondrej/Projects/pkg-php/php-src/ext/gd/tests/bug45799.php on line 3